praw/models/user.py

```diff
@@ -92,8 +92,14 @@ def friends(
         )
         return self._reddit.get(endpoint)
 
-    def karma(self) -> Dict[Subreddit, int]:
-        """Return a dictionary mapping subreddits to their karma."""
+    def karma(self) -> Dict[Subreddit, Dict[str, int]]:
+        """Return a dictionary mapping subreddits to their karma.
+
+        The returned dict contains subreddits as keys. Each subreddit key
+        contains a sub-dict that have keys for ``comment_karma`` and
+        ``link_karma``. The dict is sorted in the order of the subreddit that
+        gave you the most karma to the subreddit that gave you the least karma.
```

Comment:
I think this wording could be clarified slightly to "is sorted in decreasing karma order."

Reply:
What about `in descending karma order`?

Reply:
that works too!